e2e/tests/shared/shared.go

```diff
@@ -100,6 +100,57 @@ func SetupTestWithDefaults(testName string) (*Test, error) {
 		Provisioner:        config.Provisioner,
 	}
 
+	if utiliyManaged == "argocd" {
+		createClusterReq = &model.CreateClusterRequest{
+			AllowInstallations: true,
+			Annotations:        testAnnotations(testID),
+			AMI:                config.KopsAMI,
+			VPC:                config.VPC,
+			Provisioner:        config.Provisioner,
+			ArgocdClusterRegister: map[string]string{
+				"cluster-type": "customer",
+			},
+			DesiredUtilityVersions: map[string]*model.HelmUtilityVersion{
+				"nginx": {
+					Chart: "unmanaged",
+				},
```

Comment:
We should use the constants for all these. Example:
```suggestion
				model.NginxCanonicalName: {
					Chart: model.UnmanagedUtilityVersion,
				},
```